x-pack/plugins/security/public/account_management/user_profile/user_profile.tsx

```diff
@@ -137,6 +142,93 @@ function UserDetailsEditor({ user }: { user: AuthenticatedUser }) {
   );
 }
 
+function UserSettingsEditor({ formik }: { formik: ReturnType<typeof useUserProfileForm> }) {
+  if (!formik.values.data) {
+    return null;
+  }
+
+  return (
+    <EuiDescribedFormGroup
+      fullWidth
+      fieldFlexItemProps={{ style: { alignSelf: 'flex-start' } }}
+      title={
+        <h2>
+          <FormattedMessage
+            id="xpack.security.accountManagement.userProfile.userSettingsTitle"
+            defaultMessage="Theme"
+          />
+        </h2>
+      }
+      description={
+        <FormattedMessage
+          id="xpack.security.accountManagement.userProfile.themeFormGroupDescription"
+          defaultMessage="Select the interface theme that you wish to apply across the platform."
```

Comment:
I suggest cutting this down a bit.  Here are some suggestions:

Select the appearance of your interface.
Select the theme to apply across the interface.
Select the them to apply across the platform.
Choose between a light or dark theme, or use the default system setting. (ok this is longer, but it is more specific)

Reply:
I've switched it to the first option! Thanks for the suggestions.